api/bazel/api_build_system.bzl

```diff
@@ -188,5 +188,5 @@ def api_proto_package(
             "@com_envoyproxy_protoc_gen_validate//validate:go_default_library",
             "@com_google_googleapis//google/api:annotations_go_proto",
             "@com_google_googleapis//google/rpc:status_go_proto",
-        ],
+        ]).to_list(),
```

Comment:
How does this change relate?

Reply:
In cel library, checked.proto and syntax.proto have two proto libraries, but it merges into one go_proto. It causes build error due to duplicated dependencies. I have to add depset().to_list() to remove duplicative entry.

Reply:
Can you add a comment above this to explain to future maintainers?

Reply:
SG. I added a comment above.